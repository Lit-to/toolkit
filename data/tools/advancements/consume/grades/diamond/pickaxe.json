{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:diamond_pickaxe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/diamond/pickaxe"
    }
}