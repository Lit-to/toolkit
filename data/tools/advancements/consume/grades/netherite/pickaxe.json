{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:netherite_pickaxe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/netherite/pickaxe"
    }
}