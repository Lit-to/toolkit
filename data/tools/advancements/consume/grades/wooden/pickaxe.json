{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:wooden_pickaxe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/wooden/pickaxe"
    }
}