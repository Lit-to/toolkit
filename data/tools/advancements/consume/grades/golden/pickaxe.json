{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:golden_pickaxe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/golden/pickaxe"
    }
}